website/orders/templates/orders/order.html

```diff
@@ -26,7 +38,7 @@ <h2>You have already ordered items</h2>
         <div id="product-list">
         </div>
     </div>
-    <div class="container-sm text-center">
+    <div class="container-sm text-center pb-5">
```

Comment:
The "items in cart" should disappear in my opinion

Reply:
What is your suggestion for a replacement?

Reply:
Nothing, just remove the whole 'items in cart' since it is empty